cfme/tests/openstack/cloud/test_volumes.py

```diff
@@ -0,0 +1,53 @@
+"""Tests for Openstack cloud volumes"""
+
+import fauxfactory
+import pytest
+
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.utils import testgen
+from cfme.utils.log import logger
+from cfme.utils.wait import wait_for
+
+
+pytest_generate_tests = testgen.generate([OpenStackProvider], scope='module')
+
+pytestmark = [pytest.mark.usefixtures("setup_provider_modscope")]
+
+
+@pytest.fixture(scope='module')
+def volume_size():
+    return 1
```

Comment:
**QUESTION:** Is there a reason for this to be a fixture?

Reply:
Just wanted to keep this value in one place..

Reply:
You can just set a constant in the module and reference it in the tests.

Reply:
https://github.com/ManageIQ/integration_tests/pull/5555/files#diff-9ee724665011348e6753f3fd66392437R42

here's an example of one of my open PR's with such a change. I didn't like '4' showing up in the test methods